src/database/sqlite3/sqlite_database.cc

```diff
@@ -284,6 +284,14 @@ void Sqlite3Database::_exec(const char* query)
     exec(query, strlen(query), false);
 }
 
+std::string Sqlite3Database::quote(std::string_view value) const
+{
+    char* q = sqlite3_mprintf("'%q'", value.data());
+    std::string ret = q;
+    sqlite3_free(q);
+    return ret;
+}
+
```

Comment:
Please try to get rid of the original method

Reply:
I think this is what you meant.